GrocifyApp.UI/src/components/Products/ProductMeasuresSelector.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+
+//Internal components
+import BaseModal from '../modals/BaseModal';
+import CustomInputApp from '../CustomInputApp';
+
+//Assets & Css
+import ChevronIcon from '../../assets/chevron-filled-ic.svg';
+import styles from './SelectorDropdown.module.scss';
+
+//Consts
+import { LabelConsts } from '../../consts/ENConsts';
+import InputType from '../../consts/InputType';
+
+const ProductMeasuresSelector = ({ selectedValue, selectedValueChanged, productMeasures }) => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    const handleOpenModal = () => {
+        setIsOpen(true);
+    };
+
+    const handleCloseModal = () => {
+        setIsOpen(false);
+    };
+
+    const handleOptionChange = (measure) => {
+        selectedValueChanged(measure);
+        handleCloseModal();
+    };
+
+    return (
+        <div>
+            <div onClick={handleOpenModal}>
+                <CustomInputApp className="app-form mb-0"
+                    type={InputType.Custom}
+                    label={LabelConsts.Measure}
+                    value={selectedValue.name}
+                    onChange={selectedValueChanged}
+                    icon={ChevronIcon}
+                />
+            </div>
+
+            <BaseModal isOpen={isOpen} onClose={handleCloseModal} titleModal={LabelConsts.Measure} noFooter={true} modalBody={
+                    <div className={styles.containerMeasures}>
+                        {productMeasures.map((measure) => (
+                            <div key={measure.id} value={measure.name} onClick={() => handleOptionChange(measure)}
+                                className={`${styles.measureRow} text ${measure === selectedValue ? styles.selected : ''}`}>
```

Comment:
measure === selectedValue not working, try measure.name === selectedValue.name

also, review style padding:
![image](https://github.com/Catarina98/Grocify/assets/36730010/a0ee6f7c-5f0e-4f2c-b7b0-17dc322c1a23)